contactaccessrights/js/src/access-rights/controllers/access-rights-modal-ctrl.js

```diff
@@ -0,0 +1,113 @@
+define(['access-rights/modules/controllers'], function (controllers) {
+  'use strict';
+
+  controllers.controller('AccessRightsModalCtrl', ['Region', 'Location', 'Right', '$q', '$modalInstance',
```

Comment:
nitpicking n.3: please list first the built-in, then the vendor, then the common, finally the local dependencies
```js
['$q', '$modalInstance', 'Location', 'Region']
```